backend/setup.py

```diff
@@ -0,0 +1,65 @@
+import sqlite3 as lite
+import os
+
+
+def make_dicts(cursor, row):
+    return dict((cursor.description[idx][0], value) for idx, value in enumerate(row))
+
+#remove the database if it exists
+if os.path.exists("wellbeing.db"):
+    os.remove("wellbeing.db")
+
+#create connection to database and its cursor
+con = lite.connect("wellbeing.db", check_same_thread=False)
+con.row_factory = make_dicts
+cur = con.cursor()
+
+
+#initialize the database with wellbeing resources
+def init_db():
+    with con:
```

Comment:
i would suggest loading the .sql file, so you dont have to change them at the same time